src/meta/meta_ckpt.c

```diff
@@ -1262,6 +1262,195 @@ __wt_meta_ckptlist_update_config(
     return (ret);
 }
 
+/*
+ * __ckpt_extract_blkmod_bitmap --
+ *     Extracts the blkmod info in a checkpoint into a bitmap in a WT_ITEM.
+ */
+static int
+__ckpt_extract_blkmod_bitmap(
+  WT_SESSION_IMPL *session, WT_CKPT *ckpt, const char *id, WT_ITEM *output_bitmap)
+{
+    WT_BLOCK_MODS *blk;
+    u_int i;
+
+    WT_CLEAR(*output_bitmap);
+
+    for (i = 0, blk = &ckpt->backup_blocks[0]; i < WT_BLKINCR_MAX; ++i, ++blk)
```

Comment:
Should `&ckpt->backup_blocks[0]` be `&ckpt->backup_blocks[i]`?

Reply:
Hi @etienneptl . `&ckpt->backup_blocks[0]` is correct in this case as it initialising `blk` at the start of the loop, and then `blk` is incremented each time around. `i` is used simply to count the number of loop iterations rather than as an index.

Reply:
Thank you, I missed the {{++blk}} bit.